cli/src/pcluster/aws/ec2.py

```diff
@@ -312,8 +312,8 @@ def get_supported_az_for_instance_type(self, instance_type: str):
         """
         Return a tuple of availability zones that have the instance_type.
 
-        This function build above _get_supported_az_for_instance_types,
-        but simplify the input to 1 instance type and result to a list
```

Comment:
As far as I understood, this function returns a tuple and not a list, but feel free to correct this if it's wrong

Reply:
correct